engine/src/main/java/com/redhat/cloud/notifications/processors/camel/slack/SlackProcessor.java

```diff
@@ -29,7 +29,9 @@ protected CamelNotification getCamelNotification(Event event, Endpoint endpoint)
 
         SlackNotification notification = new SlackNotification();
         notification.webhookUrl = properties.getUrl();
-        notification.channel = properties.getExtras().get("channel");
+        if (properties.getExtras() != null &&  properties.getExtras().containsKey("channel")) {
+            notification.channel = properties.getExtras().get("channel");
+        }
```

Comment:
This looks like something related to another ticket 😄

Reply:
It can be shortened a bit:
```suggestion
        if (properties.getExtras() != null) {
            notification.channel = properties.getExtras().get("channel");
        }
```